python/lsst/atmospec/utils.py

```diff
@@ -507,3 +512,107 @@ def getLinearStagePosition(exp):
         if position is not None:
             linearStagePosition += position
```

Comment:
There is a difference of 4mm in the distance of the disperser between the hologram and the ronchi (and maybe others). At least, in Spectractor I had to write:
```
parameters.DISTANCE2CCD = 115 + float(image.header["LINSPOS"])  # mm
    if image.disperser_label == "holo4_003":
        parameters.DISTANCE2CCD += 4  # hologram is sealed with a 4 mm window
```
This has to be translated in getLinearStagePosition() function

Reply:
Line 776 of processStar.py applies this offset, I believe.  @mfisherlevine can confirm when he returns.

Reply:
Yup, that's right, thanks both!